docs/plugin/custom-executor.md

```diff
@@ -93,13 +113,28 @@ For a final implementation, see the [Botkube template repository](./quick-start.
 
    ```go
    // Execute returns a given command as a response.
-   func (EchoExecutor) Execute(_ context.Context, in executor.ExecuteInput) (executor.ExecuteOutput, error) {
+   func (*EchoExecutor) Execute(_ context.Context, in executor.ExecuteInput) (executor.ExecuteOutput, error) {
     return executor.ExecuteOutput{
       Data: fmt.Sprintf("Echo: %s", in.Command),
     }, nil
    }
    ```
 
+7. Add the required `Help` method:
+
+   ```go
+   // Help returns help message
+   func (*EchoExecutor) Help(ctx context.Context) (interactive.Message, error) {
+    return interactive.Message{
+      Base: interactive.Base{
+        Body: interactive.Body{
+          CodeBlock: "echo prints out given input string",
+        },
+      },
+    }, nil
+   }
+   ```
+
    The `Execute` method is the heart of your executor plugin. This method runs your business logic and returns the output as plain text. Next, the Botkube core sends back the response to a given communication platform.
```

Comment:
This is applicable for point no 6, so the new point no 7 is placed incorrectly. Please fix this.